### ISSUE TITLE
row instead of column?

### ISSUE THREAD

**Issue by consciousweb** [nuclearpasta/react-native-drax#111]
Is it possible to set drax for a sortable list with a wrapped flex row of items instead of a column?

**Comment by lafiosca** [comment]
Do you mean the `horizontal` prop to `DraxList` or something more complex?

**Comment by consciousweb** [comment]
@lafiosca i want it to flex wrap, so after a certain number of columns, it creates a new row

**Comment by lafiosca** [comment]
Hmm is it possible #99 could support this, if we manage to get it merged?

**Comment by consciousweb** [comment]
It would certainly be an improvement. Otherwise the ability to directly
style the list would at least be helpful

**Comment by lafiosca** [comment]
Any props you pass to `DraxList` that it does not consume are passed to the underlying `Flatlist`. I’m not sure if that helps.

**Comment by consciousweb** [comment]
nothing i pass to the style prop of draxlist is reflected at render.

**Comment by lafiosca** [comment]
Yes I suppose that makes sense if the DraxList is building its own style. (I apologize, but I’ve been making these short replies from away from the computer without an opportunity to dig into the code.)

I’ve never seen a wrapped flex row list before and didn’t realize FlatList supports it. But thinking about it some more, it would definitely require DraxList to support x/y shift offsets rather than just a single dimension like it does now. So I think a PR like the one I linked above would be a prerequisite before we could make this work, and then an additional enhancement would be required to allow you to pass the proper style parameters. It seems to me it should be possible to pass your own styles and have them merge with and/or override whatever DraxList is using, at the user’s own risk. For now I will leave this issue open as an enhancement request.

I have recently started a new job and am unsure how soon I will be putting any significant dev sessions into Drax, but other contributors are also welcome.
